### PR TITLE
Update cms-element-form-privacy.html.twig - make privacyUrl and tosUr…

### PR DESCRIPTION
**Pull request by bitranox**
Shopware Issue tracker Ticket: NEXT-17851 - privacyUrl and tosUrl not working
### 1. Why is this change necessary?
privacyUrl and tosUrl not working

### 2. What does this change do, exactly?
fix the Error

### 4. Please link to the relevant issues (if any).
Shopware Issue tracker Ticket: NEXT-17851

### CODE REVIEW

File: src/Storefront/Resources/views/storefront/element/cms-element-form/form-components/cms-element-form-privacy.html.twig [hunk 1]
```diff
@@ -17,9 +17,14 @@
         {% block cms_form_privacy_opt_in_label %}
             <label for="{{ identifierTemplate|format(_key) }}" class=" custom-control-label">
                 {{ "general.privacyNotice"|trans({
-                    '%url%': path('frontend.cms.page',{ id: config('core.basicInformation.privacyPage') })
```
Comment (shyim) [inline src/Storefront/Resources/views/storefront/element/cms-element-form/form-components/cms-element-form-privacy.html.twig:20]: We need still to replace %url% as someone should adjusted that to that value in the translation

**Comment by shyim** [comment]
Hey,

Can you add a changelog file according to this ADR? https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-Implement-New-Changelog.md

**Comment by bitranox** [comment]
ok, it still complains about "You should not use the same commit message multiple times" - 
and I can not remove that. Someone can take care please, i am finished.

**Comment by NiklasLimberg** [comment]
Hi @bitranox,
I'll take care of this PR, consider it merged

**Comment by shyim** [comment]
Thanks for your Contribution! :blue_heart:

**Pull request closed without merging**